app/controllers/nominations/nominate_induction_coordinator_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class Nominations::NominateInductionCoordinatorController < ApplicationController
+  def new
+    @token = params[:token]
+    @nomination_email = NominationEmail.find_by(token: @token)
```

Comment:
Out of interest - should we encrypt the token in the database? It feels a bit like we're storing plain-text passwords here

Reply:
I think that would be overkill and I don't think it would be any more secure than what we currently have. The issue would still be that anyone who gets the email and clicks on the link can claim the school even if we add encryption.

Reply:
The problem is when attacker manages in any way to read the database. In this case, as the tokens are not hashed, he automatically can impersonate anyone we send those tokens to. Naturally it is not as bad as having un-hashed passwords (or, as mentioned in the link below - password reset tokens) as it will only give an access to limited part of the site. But, hackers are freakishly clever. Hashing one-time tokens is pretty much a security standard now-a-days, Devise does it automatically for all their token-led emails (reset, confirmation, invitations etc)

https://security.stackexchange.com/questions/86913/should-password-reset-tokens-be-hashed-when-stored-in-a-database